scripts/generate_protobuf_and_funcs.rb

```diff
@@ -441,6 +441,39 @@ def generate!
   // Named tokens in scan.l
   #{@scan_protobuf_tokens.join("\n  ")}
 }
+
+message SummaryResult {
+  message SummaryResultTable {
```

Comment:
In general, do y'all have a preference for nested message types, or non-nested?

If you do have a preference, is it just a style thing, or is there a notable difference (e.g. in performance or something)?

Reply:
I don't believe there's a performance difference, but one benefit to nesting messages is you can use a less-specific message name, e.g. renaming `SummaryResultTable` to `Table`, without running into naming conflicts from other messages.